### PR TITLE
Signaled correctly when rev query param is not provided on deprecation

### LINKED ISSUE #51: Deprecate a project without a rev should return the correct response
HTTP response: Bad request
HTTP metadata
```json
{
  "@context" : "http://127.0.0.1:8080/v1/contexts/nexus/core/error/v0.1.0",
  "code" : "MissingParameter",
  "message" : "'rev' parameter is required"
}

```

### PR DESCRIPTION
**Pull request by umbreak**
Fixes #51

### CODE REVIEW

File: modules/service/src/main/scala/ch/epfl/bluebrain/nexus/admin/service/handlers/ExceptionHandling.scala [hunk 1]
```diff
@@ -67,7 +67,10 @@ class ExceptionHandling(implicit errorContext: ContextUri) {
     }
 
   private implicit val commonFrom: StatusFrom[CommonRejections] =
-    StatusFrom(_ => BadRequest)
+    StatusFrom {
```
Comment (bogdanromanx) [inline modules/service/src/main/scala/ch/epfl/bluebrain/nexus/admin/service/handlers/ExceptionHandling.scala]: we should keep badrequest for missing expected query parameters.

File: modules/service/src/main/scala/ch/epfl/bluebrain/nexus/admin/service/handlers/ExceptionHandling.scala [hunk 2]
```diff
@@ -67,7 +67,10 @@ class ExceptionHandling(implicit errorContext: ContextUri) {
...
+      case _: MissingParameter => Conflict
```
Comment (wwajerowicz) [inline modules/service/src/main/scala/ch/epfl/bluebrain/nexus/admin/service/handlers/ExceptionHandling.scala]: I think missing parameter should be a `Bad Request` not `Conflict`.

File: modules/service/src/main/scala/ch/epfl/bluebrain/nexus/admin/service/handlers/RejectionHandling.scala [hunk 3]
```diff
@@ -38,6 +38,8 @@ object RejectionHandling {
           complete(BadRequest -> (IllegalParam(err.getMessage): CommonRejections))
         case ValidationRejection(_, Some(e: IllegalPayload)) =>
           complete(BadRequest -> (e: CommonRejections))
+        case MissingQueryParamRejection(param) =>
+          complete(Conflict -> (MissingParameter(s"'$param' parameter is required"): CommonRejections))
```
Comment (bogdanromanx) [inline modules/service/src/main/scala/ch/epfl/bluebrain/nexus/admin/service/handlers/RejectionHandling.scala]: same as above

Comment (wwajerowicz) [inline modules/service/src/main/scala/ch/epfl/bluebrain/nexus/admin/service/handlers/RejectionHandling.scala]: same here.

**Pull request merged**